stock_storage_type/models/stock_location.py

```diff
@@ -153,6 +153,11 @@ def _should_compute_will_contain_lot_ids(self):
             for location in self.allowed_location_storage_type_ids
         )
 
+    def _should_compute_is_empty(self):
+        return self.usage == "internal" and any(
+            location.only_empty for location in self.allowed_location_storage_type_ids
```

Comment:
does this flag exists on location? :thinking:

Reply:
@simahawk The same error exists in `_should_compute_will_contain_lot_ids`  and `_should_compute_will_contain_product_ids`. I'll modify the code for all these methods

Reply:
@simahawk Change done